Day2.hs

```diff
@@ -0,0 +1,71 @@
+module Day2
+( t1
+, t2
+, main
+) where
+import qualified Data.Text as Text
+import qualified Data.Text.IO as Text
+import qualified Data.Text.Read as Text
+import Control.Monad (unless)
+import Data.Char (isDigit, isLetter)
+import Data.List (elemIndices)
+import qualified Text.ParserCombinators.ReadP as ReadP
```

Comment:
Bold choice to use the built-in parser combinators, but nothing wrong with that :+1:

Reply:
Awesome! Out of curiosity: Which library would have been your choice?

Reply:
https://hackage.haskell.org/package/megaparsec or https://hackage.haskell.org/package/trifecta, probably.